src/components/attendee/RetunableMediaElement/RetunableMediaElement.tsx

```diff
@@ -1,4 +1,4 @@
-import { useCallback, useMemo, useState } from "react";
+import React, { useCallback, useMemo, useState } from "react";
```

Comment:
[NOTE] Same as before, React is no longer required as an import, no need for the default import.

Reply:
The rest are fine, it would look weird to use `React.useMemo` or `React.useCallback`